noxfile.py

```diff
@@ -162,15 +162,24 @@ def conftest_retry(session):
     if not conformance_test_exists and not conformance_test_folder_exists:
         session.skip("Conformance tests were not found")
 
-    session.install("pytest",)
+    # Install all test dependencies and pytest plugin to run tests in parallel.
+    # Then install this package in-place.
+    session.install("pytest", "pytest-xdist")
     session.install("-e", ".")
 
     # Run py.test against the conformance tests.
     if conformance_test_exists:
-        session.run("py.test", "--quiet", conformance_test_path, *session.posargs)
+        session.run(
+            "py.test", "-n", "auto", "--quiet", conformance_test_path, *session.posargs
+        )
```

Comment:
I think it might be a better DX to inject the `"-n", "auto"` into the command line only if `session.posargs` is empty:  that way, the user could chose to e.g. restrict the number of cores available, or even distribute the tests across multiple hosts.

Reply:
@tseaver good idea! I've updated the noxfile. PTAL, thanks!